src/sidebar/services/session.js

```diff
@@ -58,8 +58,8 @@ export class SessionService {
       // the /app endpoint.
       this._lastLoadTime = Date.now();
       this._lastLoad = retryPromiseOperation(() => {
-        const opts = this._authority ? { authority: this._authority } : {};
-        return this._api.profile.read(opts);
+        const authority = this._authority;
+        return this._api.profile.read(authority ? { authority } : {});
```

Comment:
Can you help me understand how these changes relate to the other changes (I mean, this seems fine and dandy, I'm just wondering what prompted it)?

Reply:
I should have added a comment. There is no functional change here. This just helps TypeScript understand better that `read` is called either with an `authority` param that is a string, or with no parameters. Before it incorrectly deduced that it might have been called with an `authority` param that was `undefined`.